packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -496,177 +496,187 @@ export default class TableResize implements EditorPlugin {
-                                    }
-                                } else {
-                                    cell.td.style.height = '';
+                                    : cell.td.getBoundingClientRect().bottom -
+                                      cell.td.getBoundingClientRect().top;
+                                const newHeight = originalHeight * ratioY;
+                                if (newHeight >= MIN_CELL_HEIGHT) {
+                                    cell.td.style.height = `${newHeight}px`;
                                 }
+                            } else {
+                                cell.td.style.height = null;
                             }
                         }
                     }
                 }
             }
-            rect = normalizeRect(this.currentTable.getBoundingClientRect());
-            currentBorder = this.isRTL ? rect.left : rect.right;
-            this.currentTable.setAttribute(
-                this.isRTL ? 'currentLeftBorder' : 'currentRightBorder',
-                currentBorder.toString()
-            );
+        }
+        rect = normalizeRect(vtable.table.getBoundingClientRect());
+        currentBorder = this.isRTL ? rect.left : rect.right;
+        vtable.table.setAttribute(
+            this.isRTL ? 'currentLeftBorder' : 'currentRightBorder',
+            currentBorder.toString()
+        );
 
-            const currentBottomBorder = this.currentTable.getBoundingClientRect().bottom;
-            this.currentTable.setAttribute('currentBottomBorder', currentBottomBorder.toString());
-            vtable.writeBack();
+        const currentBottomBorder = vtable.table.getBoundingClientRect().bottom;
+        vtable.table.setAttribute('currentBottomBorder', currentBottomBorder.toString());
+        vtable.writeBack();
+        return;
+    };
+
+    private resizeRows = (newPos: number, rect: Rect, vtable: VTable) => {
+        if (!this.isReadyForResizingHeight) {
+            preProcessTable(vtable.table, ResizeState.Horizontal);
+            this.isReadyForResizingHeight = true;
+        }
+        vtable.forEachCellOfCurrentRow(cell => {
+            if (cell.td) {
+                cell.td.style.height = cell.td == this.currentTd ? `${newPos - rect.top}px` : null;
+            }
+        });
+    };
+
+    private resizeColumns = (newPos: number, vtable: VTable, isShiftPressed: boolean) => {
+        if (!this.isReadyForResizingWidth) {
+            preProcessTable(vtable.table, ResizeState.Vertical);
+            this.isReadyForResizingWidth = true;
+        }
+        // Since we allow the user to resize the table width on adjusting the border of the last cell,
+        // we need to make the table width resizeable by setting it as null;
+        // We also allow the user to resize the table width if Shift key is pressed
+        if (!this.currentCellsToResize[0].nextElementSibling || isShiftPressed) {
```

Comment:
Better check if current column number is the last one in table, but not checking nextElementSibling

Reply:
fixed